bzip2-sys/build.rs

```diff
@@ -23,6 +23,29 @@ fn main() {
         }
     }
 
+    // List out the WASM targets that need wasm-shim.
+    // Note that Emscripten already provides its own C standard library so
+    // wasm32-unknown-emscripten should not be included here.
+    // See: https://github.com/gyscos/zstd-rs/pull/209
+    let need_wasm_shim = env::var("TARGET").map_or(false, |target| {
+        target == "wasm32-unknown-unknown" || target == "wasm32-wasi"
```

Comment:
WASI shouldn't need a shim I believe since that should use the wasi-sdk C compiler which has all the support necessary.

I believe this PR should only affect wasm32-unknown-unknown